src/libraries/System.Text.Json/src/System/Text/Json/Serialization/WriteStack.cs

```diff
@@ -347,33 +351,37 @@ public async ValueTask DisposePendingDisposablesOnExceptionAsync()
         {
             Exception? exception = null;
 
-            exception = await DisposeFrame(Current.CollectionEnumerator, Current.AsyncDisposable, exception).ConfigureAwait(false);
+            exception = await DisposeFrame(Current.CollectionEnumerator, Current.AsyncDisposable, Current.Disposable, exception).ConfigureAwait(false);
```

Comment:
I don't think anything needs to change in these methods.

Reply:
Removed, as a consequence of the previous comment.